docs/Setup/NodeSDK-setup.md

```diff
@@ -0,0 +1,253 @@
+      // 'userName', and determined that we need to invoke the chaincode
+      // with 'chaincodeID' and function named 'fcn' with arguments 'args'.
+      handleUserRequest(userName,chaincodeID,fcn,args);
+   }
+}
+
+// Handle a user request
+function handleUserRequest(userName, chaincodeID, fcn, args) {
+   // Register and enroll this user.
+   // If this user has already been registered and/or enrolled, this will
+   // still succeed because the state is kept in the KeyValStore
+   // (i.e. in '/tmp/keyValStore' in this sample).
+   var registrationRequest = {
+        enrollmentID: userName,
+        // Customize account & affiliation
+        account: "bank_a",
+        affiliation: "00001"
+   };
+   chain.registerAndEnroll( registrationRequest, function(err, user) {
+      if (err) return console.log("ERROR: %s",err);
+      // Issue an invoke request
+      var invokeRequest = {
+        // Name (hash) required for invoke
+        chaincodeID: chaincodeID,
+        // Function to trigger
+        fcn: fcn,
+        // Parameters for the invoke function
+        args: args
+     };
+     // Invoke the request from the user object.
+     var tx = user.invoke(invokeRequest);
+     // Listen for the 'submitted' event
+     tx.on('submitted', function(results) {
+        console.log("submitted invoke: %j",results);
+     });
+     // Listen for the 'complete' event.
+     tx.on('complete', function(results) {
+        console.log("completed invoke: %j",results;
+     });
+     // Listen for the 'error' event.
+     tx.on('error', function(err) {
+        console.log("error on invoke: %j",err);
+     });
+   });
+}
+```
+
+## Getting Set Up
+
+First, you'll want to have a running peer node and CA. You can follow the instructions for setting up a network [here](../../docs/Setup/ca-setup.md#Operating-the-CA), and start a single peer node and CA.
+
+### Chaincode Deployment Directory Structure
+
+To have the chaincode deployment succeed in network mode, you must properly set up the chaincode project outside of your Hyperledger Fabric source tree. These instructions will demonstrate how to properly set up the directory structure to deploy *chaincode_example02* in network mode.
+
+The chaincode project must be placed inside the `src` directory in your local `$GOPATH`. For example, the [chaincode_example02](https://github.com/hyperledger/fabric/blob/master/examples/chaincode/go/chaincode_example02/chaincode_example02.go) project may be placed inside `$GOPATH/src/` as shown below.
+
+```
+mkdir -p $GOPATH/src/github.com/chaincode_example02/
+cd $GOPATH/src/github.com/chaincode_example02
+curl GET https://raw.githubusercontent.com/hyperledger/fabric/master/examples/chaincode/go/chaincode_example02/chaincode_example02.go > chaincode_example02.go
+```
+
+Once you have placed your chaincode project inside the `src` directory in your local `$GOPATH`, you will need to vendor the dependencies. In your chaincode source directory, run the following commands:
```

Comment:
*From the directory containing* your chaincode source, *execute* the following commands: